Recruitment/JobSpecs/why_you_js_engineer.md

```diff
@@ -1,43 +1,19 @@
-### Why You?
+### What you'll do
+* Work intimiately with clients to understand their organisation, aspirations and challenges.
+* Help clients identify promising solutions through participating in, and leading, digital workshops.
+* Build real solutions through experimentation as part of a multi-faceted team.  
+* Accelerate and improve how we help our clients through developing and contributing to components, services and methodologies.
```

Comment:
So maybe we need to say something like "we do great new things, using great new tech, but we don't always work with clients, lots of cool things to build internally as well. and contribute back to the community. peace."